### ISSUE TITLE
Stop non-existent method/function in remove_action()

### ISSUE THREAD

**Issue by szepeviktor** [szepeviktor/phpstan-wordpress#197]
```php
remove_action('wp_enqueue_scripts', [$theme, 'wp_common_block_scripts_and_styles']);
```

This triggered no error on level 5.
How could that be?

**Comment by johnbillion** [comment]
This is expected behaviour. The `$callback` parameter doesn't have to be valid.

```php
 * @param callable|string|array $callback  The name of the function which should be removed.
 *                                         This function can be called unconditionally to speculatively remove
 *                                         a callback that may or may not exist.
```

**Comment by szepeviktor** [comment]
Thank you, John.

(For extraterrestrial aliens 👽) could you add something like `HookCallbackRule` that checks for callback existence in hooking function?

**Comment by johnbillion** [comment]
I'd rather not 🤣
